src/lib/generateDocument/generateDocument.ts

```diff
@@ -69,7 +69,10 @@ export default function generateClientIdDocument({
     contacts: contact ? [contact] : [],
     application_type: applicationType,
     require_auth_time: requireAuthTime,
-    default_max_age: defaultMaxAge,
+    default_max_age:
+      typeof defaultMaxAge === "string"
+        ? Number(defaultMaxAge) || undefined
+        : defaultMaxAge,
```

Comment:
Is it possible to error out here, and prompt the user for a correct input if the string doesnt parse to a number ?

Reply:
I solved this by defaulting to the incorrect string value. This will be caught by the form validation.

Reply:
I'm not sure I get what you mean. If `Number(defaultMaxAge)` returns `NaN`, `default_max_age` becomes undefined, which is a correct value, but probably not what the user intended. Or am I missing something?